Classes/Command/FluidTemplatePrecompileCommand.php

```diff
@@ -0,0 +1,163 @@
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Precompile Fluid templates')
+            ->addArgument(
+                'extension',
+                InputArgument::OPTIONAL,
+                'Extension key of which the fluid template should be compiled, leave empty to compile all',
+                null
+            )
+            ->addOption(
+                'fail',
+                null,
+                InputOption::VALUE_NONE,
+                'Fail with a non-zero return code if any template could not be compiled'
+            )
+            ->addOption(
+                'show-only-failed',
+                null,
+                InputOption::VALUE_NONE,
+                'Show only failed fluid templates from compiling. Only in conjunction with -v'
+            )
+            ->addOption(
+                'limit',
+                'l',
+                InputOption::VALUE_REQUIRED,
+                'Limit of template files of each extension shown. Only in conjunction with -v',
+                5
+            )
+            ->addOption(
+                'disable-error-handler',
+                null,
+                InputOption::VALUE_NONE,
+                'Temporary disable any registered error handler for compiling'
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $outputHelper = new SymfonyStyle($input, $output);
+
+        // Get arguments and options
+        $extension = $input->getArgument('extension');
+        $fail = $input->getOption('fail');
+        $onlyFailed = $input->getOption('show-only-failed');
+        $limit = $input->getOption('limit');
+        $disableErrorHandlers = $input->getOption('disable-error-handler');
+
+        // Test exception handlers
+        if (set_error_handler(function() {}) && !$disableErrorHandlers) {
+            $outputHelper->warning(
+                "\nIt seems like there is an error handler registered."
+                . "\nIn some cases this can prevent the precompiling to run properly."
+                . "\nAn ErrorHandler might raise an exception for a PHP Warning."
+            );
+
+            restore_error_handler();
+        }
+
+        $result = $this->getPrecompilerService()->warmup($extension);
+
+        // General information
+        $templatesCount = 0;
+        foreach ($result as $extensionResult) {
+            $templatesCount += count($extensionResult);
+        }
+
+        $uncompilable = array_column($result, 'uncompilable');
+        $uncompilable = array_sum($uncompilable);
+
+        $outputHelper->section('General');
+        $outputHelper->text(vsprintf('Compiled %d extension(s) with %d template(s) in total.', [count($result), $templatesCount]));
+        $outputHelper->text(vsprintf('%d template(s) could not be compiled.', [$uncompilable]));
```

Comment:
This should be shown only if necessary and as warning/error.

Reply:
I see this as an important information, even when no verbose output is requested and the command should not fail.
You may want to warmup your caches, including the Fluid resources, and have some nice metric.

I'm also against the `-v` option because this also shows the detailed compilation report where you can already extract this metric.

How do you think about this?

Reply:
It's vital information and should be shown. The trick is to only error out if failures are wanted - but the information that you do have uncompilable templates is pretty important, I agree.